### DISCUSSION TITLE
Is there a way to see stats for operator-only repos?

### DISCUSSION THREAD

**Discussion by marshtompsxd** [artifacthub/hub#3800] [Q&A]
Hi, thanks for building and maintaining the awesome artifact hub. I am wondering whether I can the stat data https://artifacthub.io/stats for operator-only repos? Such as the number of registered operator repos over years.

**Comment by tegioz** [comment]
Hi @marshtompsxd 👋

Thank you, we're glad you like it! 🙂

No, unfortunately all the stats we generate are across all artifact kinds, not per kind or using any filter.
